data/learnsets.ts

```diff
@@ -99465,30 +99465,74 @@ export const Learnsets: import('../sim/dex-species').LearnsetDataTable = {
 	},
 	chuggalong: {
 		learnset: {
-			clangingscales: ["9L1"],
-			clangoroussoul: ["9L1"],
-			destinybond: ["9L1"],
+			acidspray: ["9M"],
+			bite: ["9L1"],
+			bodyslam: ["9M"],
+			bulldoze: ["9M"],
+			celebrate: ["9S0"],
+			clangingscales: ["9L52"],
+			clangoroussoul: ["9L64"],
+			crunch: ["9M", "9L32"],
+			destinybond: ["9E"],
 			dracometeor: ["9M"],
-			dragontail: ["9M"],
+			dragonbreath: ["9L12"],
+			dragoncheer: ["9M"],
+			dragonclaw: ["9M"],
+			dragondance: ["9M", "9L1", "9S0"],
+			dragonpulse: ["9M", "9L36"],
+			dragonrush: ["9E"],
+			dragontail: ["9M", "9L1", "9S0"],
+			earthquake: ["9M"],
 			encore: ["9M"],
 			endure: ["9M"],
 			facade: ["9M"],
 			flamethrower: ["9M"],
-			healbell: ["9L1"],
+			flashcannon: ["9M"],
+			gigaimpact: ["9M"],
+			gunkshot: ["9M", "9L58"],
+			healbell: ["9E"],
+			heavyslam: ["9M"],
 			helpinghand: ["9M"],
-			protect: ["9M"],
+			hyperbeam: ["9M"],
+			irondefense: ["9M", "9L46"],
+			ironhead: ["9M"],
+			lastresort: ["9E"],
+			metalsound: ["9M"],
+			nobleroar: ["9E"],
+			outrage: ["9M"],
+			poisongas: ["9L1"],
+			poisonjab: ["9M"],
+			poisontail: ["9M"],
+			protect: ["9M", "9L24"],
+			raindance: ["9M"],
 			rest: ["9M"],
-			roar: ["9M"],
+			roar: ["9M", "9L28"],
+			rockslide: ["9M"],
+			rocktomb: ["9M"],
+			scaleshot: ["9M"],
+			scaryface: ["9M"],
 			sleeptalk: ["9M"],
-			sludgebomb: ["9M"],
+			sludge: ["9L20"],
+			sludgebomb: ["9M", "9L41", "9S0"],
 			sludgewave: ["9M"],
+			smog: ["9L1"],
+			snarl: ["9M"],
+			stompingtantrum: ["9M"],
 			storedpower: ["9M"],
 			substitute: ["9M"],
+			sunnyday: ["9M"],
 			surf: ["9M"],
+			tackle: ["9L1"],
 			takedown: ["9M"],
-			taunt: ["9M"],
+			taunt: ["9M", "9L12"],
 			terablast: ["9M"],
+			trailblaze: ["9M"],
+			uproar: ["9M"],
+			venoshock: ["9M"],
 		},
+		eventData: [
+			{generation: 9, level: 50, shiny: true, abilities: ["armortail"],  moves: ["celebrate", "dragontail", "sludgebomb", "dragondance"], pokeball: "cherishball"},
```

Comment:
```suggestion
			{generation: 9, level: 50, shiny: true, abilities: ["armortail"], moves: ["celebrate", "dragontail", "sludgebomb", "dragondance"], pokeball: "cherishball"},
```

Reply:
Fixed (Github's "Commit suggestion" button is bugged)